app/discovery/discovery.go

```diff
@@ -36,6 +36,8 @@ type URLMapper struct {
 
 	AssetsLocation string
 	AssetsWebRoot  string
+
+	dead bool
```

Comment:
I'd rather flip it and call alive, as it will prevent confusing !dead checks

Reply:
Actually I thought about it but decided to name it _dead_. To me, the destination is by default alive. At least before first health check is completed. If user decided to not use this optional functionality at all, all destinations  should be marked as alive all the time. Default value of _dead_ `false` fits here well.
Having named it as _alive_  means to set this value to `true` every time new URLMapper is created. I prefer to have reasonable default value than to worry if URLMapper correctly initialised. What do you think?

Reply:
from this pov it makes sense. having proper default overweight the potential inversed check